crates/consensus/src/transaction/eip1559.rs

```diff
@@ -296,7 +296,7 @@ impl Transaction for TxEip1559 {
         self.max_priority_fee_per_gas
     }
 
-    fn to(&self) -> TxKind {
+    fn kind(&self) -> TxKind {
         self.to
     }
```

Comment:
can we add a to function that does `kind().to()`

https://github.com/paradigmxyz/reth/blob/5e386130da0b123e107f5eb5a9a3a3622bc50129/crates/primitives/src/transaction/mod.rs#L241-L247

Reply:
sure, good idea, done

Reply:
In next release, we will be able to rm the two old functions in reth :)